main.go

```diff
@@ -64,18 +70,12 @@ func checkoutNewBranch(id, summary string) error {
 	branchName := fmt.Sprintf("%s--%s", id, dashed)
 	fmt.Println(branchName)
 	// Could maybe stash here first?
-	cmd := exec.Command("git", "checkout", "-b", branchName)
+	cmd := exec.Command("git", "checkout", "-b", branchName) // This could probably be replaced with something like `git-go` to avoid shelling out
```

Comment:
I had git-go in use at first, but decided this was simpler and fine for now.

One thing I would like is to better capture the git output - e.g. right now if the checkout fails, it just returns `exit status 1` rather than the message from git. I tried to pipe the stdout and stderr but still wound up with the unhelpful message. I'm probably doing something wrong.

Reply:
```go
cmd := exec.Command("git", "checkout", "-b", branchName)
cmd.Stdout = os.Stdout // This is the line you're missing
cmd.Run()
```